publication-commons/src/main/java/no/unit/nva/publication/model/business/logentry/FileLogEntry.java

```diff
@@ -31,6 +32,7 @@ public static final class Builder {
         private String fileType;
         private Instant timestamp;
         private LogUser performedBy;
+        private ImportSource importSource;
```

Comment:
is it tricky to reuse LogUser? It seems a bit off that all filelogEntries should have importSource